src/universal/redux/makeReducer.js

```diff
@@ -1,32 +1,8 @@
 import {combineReducers} from 'redux'
-import {reducer as formReducer, actionTypes} from 'redux-form'
 import makeRootReducer from 'universal/redux/rootDuck'
 
-const {SET_SUBMIT_SUCCEEDED} = actionTypes
-
-// wipe the value clean when submitted
-const wipeAfterSuccess = (formToClear) => (state, action) => {
-  if (action.type === SET_SUBMIT_SUCCEEDED) {
-    if (action.meta.form === formToClear) {
-      return undefined
-    }
-  }
-  return state
-}
-
-const formPluginFactory = () => {
-  // add new fields to this array if you want em cleared
-  const clearMeAfterSubmit = ['inviteTeamMember']
-  return clearMeAfterSubmit.reduce((formPlugin, name) => {
-    formPlugin[name] = wipeAfterSuccess(name)
-    return formPlugin
-  }, {})
-}
-
-const formPlugin = formPluginFactory()
-
 const appReducers = {
-  form: formReducer.plugin(formPlugin)
+  noop: (state = null) => state
```

Comment:
Since this is a big noop now, why keep this file?

Reply:
we still use redux for the kanban filtering, as soon as that goes away we can nuke this sucker